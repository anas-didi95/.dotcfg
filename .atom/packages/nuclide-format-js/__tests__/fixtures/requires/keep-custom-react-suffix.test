const Y = require('Y.mamlas');

const x = <Y />;
