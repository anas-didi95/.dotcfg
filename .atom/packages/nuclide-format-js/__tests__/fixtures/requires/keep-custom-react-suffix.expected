const React = require('React');
const Y = require('Y.mamlas');

const x = <Y />;
